.github/workflows/windows-package.yml

```diff
@@ -29,3 +33,7 @@ jobs:
     - run: pacman -Ql mingw-w64-x86_64-openblas
     - run: pacman -Ql mingw-w64-x86_64-gcc-libgfortran
     - run: ls -l /mingw64/
+    - run: python -m pip install -r requirements_dev.txt
```

Comment:
Looks like it is not finding python. Can you please run this command before any of the python commands
`export PATH=`cygpath "${pythonLocation}"`:$PATH`

I found this solution here:
https://github.com/actions/setup-python/issues/186#issuecomment-775936692

also, please combine all run commands into a single run statement like you have done before in python-package.yml

Reply:
ok